bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/details/LineItemDetailsFragment.kt

```diff
@@ -225,9 +245,20 @@ class LineItemDetailsFragment : Fragment(), LineItemDetailsScreenContract.View,
      */
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        addGlobalFocusChangeListener()
         setInputHandlers()
     }
 
+    private fun addGlobalFocusChangeListener() {
+        // Workaround for the bug in TextInputEditText where the focus change listener is not called
+        // when using end icons: https://github.com/material-components/material-components-android/issues/1015
+        binding.root.viewTreeObserver.addOnGlobalFocusChangeListener { oldFocus, _ ->
+            if (oldFocus == binding.quantity) {
+                presenter?.onQuantityInputFieldFocusLoss(binding.quantity.text.toString())
+            }
```

Comment:
It's still a bit not clear what is it the focus change here. Please see my video attached. It's related to the focus change 

https://user-images.githubusercontent.com/3911816/179530868-7473ea70-6205-4f79-bfb9-5028aadbfb07.mp4

Reply:
I can make the font size decrease to a minimum. Won't guarantee that it will always fit, but for the majority of cases it should be ok.

Reply:
> I can make the font size decrease to a minimum. Won't guarantee that it will always fit, but for the majority of cases, it should be ok.

Could be great 👍